server/sonar-server/src/main/java/org/sonar/server/es/textsearch/ComponentTextSearchQueryFactory.java

```diff
@@ -90,7 +90,7 @@ private ComponentTextSearchQuery(Builder builder) {
       this.favoriteKeys = builder.favoriteKeys;
     }
 
-    private static List<String> split(String queryText) {
+    public static List<String> split(String queryText) {
```

Comment:
Should the class be renamed ? "Component" does not sound relevant.

Reply:
I moved that method to a new helper class.